### ISSUE TITLE
Docs, Cluster Specification: incorrect convergence remark

### ISSUE THREAD

**Issue by baltiyskiy** [akka/akka#26530]
https://doc.akka.io/docs/akka/current/common/cluster.html#leader-actions states this:

```
weakly up -> up (no convergence is required for this leader action to be performed)
```

This contradicts the text a few paragraphs above, which states that this transition can only be done when convergence is reached:

```
By enabling akka.cluster.allow-weakly-up-members (enabled by default) it is possible to let new joining nodes be promoted while convergence is not yet reached. These Joining nodes will be promoted as WeaklyUp. Once gossip convergence is reached, the leader will move WeaklyUp members to Up.
```

Apparently, two actions: `Joining -> WeaklyUp` and `WeaklyUp -> Up` have been confused in one.

**Comment by patriknw** [comment]
Thanks for noticing, indeed wrong. I think it should be two items:

* joining -> weakly up (no convergence is required for this leader action to be performed)
* weakly up -> up

**Comment by patriknw** [comment]
Would you be able to contribute with a pull request fixing the issue?

**Comment by baltiyskiy** [comment]
I'd love to

**Comment by jewertow** [comment]
@patriknw issue resolved in #28306

**Comment by patriknw** [comment]
thanks
